src/styles/_DescriptionText.scss

```diff
@@ -0,0 +1,33 @@
+@use 'variables' as v;
+@use 'mixins' as M;
```

Comment:
Small change: this mixin is being aliased as "M" and the rest are "m"

Reply:
Noted